src/python-common/ceph/tests/test_service_spec.py

```diff
@@ -109,3 +109,51 @@ def test_servicespec_map_test(s_type, o_spec, s_id):
     assert spec.validate() is None
     ServiceSpec.from_json(spec.to_json())
```

Comment:
Is this required if we are not testing it?

Reply:
cause `ServiceSpec` does not yet implement `__eq__`. there is a different PR for that.